app/controllers/user/lessons_controller.rb

```diff
@@ -13,6 +16,13 @@ def show
                                            component: @lesson_test).count
   end
 
+  # GET user/courses/:course_id/lessons/:id/study
+  def study
+    @course = @lesson&.course
```

Comment:
```suggestion
    @course = @lesson.course
```
Có check rồi nên k cần toàn tử an toàn